### PR TITLE
[SPARK-21498][Examples] quick start -> one py demo have some bug in code

### PR DESCRIPTION
**Pull request by lizhaoch**
I find a bug about 'quick start',and created a new issues,Sean Owen  let
me to make a pull request, and I do

## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)
(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

**Comment by jerryshao** [comment]
Can you please fix the PR title, also update the PR description to be more meaningful.

File: docs/quick-start.md [hunk 1]
```diff
@@ -389,7 +389,7 @@ As an example, we'll create a simple Spark application, `SimpleApp.py`:
 from pyspark.sql import SparkSession
 
 logFile = "YOUR_SPARK_HOME/README.md"  # Should be some file on your system
-spark = SparkSession.builder().appName(appName).master(master).getOrCreate()
+spark = SparkSession.builder.appName("SimpleApp").getOrCreate()
```
Comment (HyukjinKwon) [inline docs/quick-start.md]: Not a big deal at all but could we remove the trailing space at the end?

**Review by HyukjinKwon — APPROVED** [review]
LGTM

File: docs/quick-start.md (hunk 1 above)
Comment (felixcheung) [inline docs/quick-start.md]: what about master?

Comment (HyukjinKwon) [inline docs/quick-start.md]: I guess that should be fine.



it is described below and this looks doing the same thing with equivalent Scala / Java examples here.

**Review by HyukjinKwon — COMMENTED** [review]
While we are here, would you mind fixing 

- 424L - `` (e.g. `pip instal pyspark` `` -> `` (e.g., `pip install pyspark`) ``

- 424L - `spark-submit` -> `` `spark-submit` ``

- 427L - `# Use spark-submit to run your application` -> `# Use Python interpeter to run your application` for Python interpreter example

**Comment by lizhaoch** [comment]
OK，I have done

File: docs/quick-start.md [hunk 2]
```diff
@@ -421,16 +421,15 @@ $ YOUR_SPARK_HOME/bin/spark-submit \
...
 
 {% highlight bash %}
-# Use spark-submit to run your application
+# Use the Python Interpreter to run your application
 $ python SimpleApp.py
 ...
 Lines with a: 46, Lines with b: 23
 {% endhighlight %}
 
-
```
Comment (HyukjinKwon) [inline docs/quick-start.md]: Let's revert removing a newline here. Sounds unrelated.

File: docs/quick-start.md [hunk 3]
```diff
@@ -421,16 +421,15 @@ $ YOUR_SPARK_HOME/bin/spark-submit \
 Lines with a: 46, Lines with b: 23
 {% endhighlight %}
 
-If you have PySpark pip installed into your enviroment (e.g. `pip instal pyspark` you can run your application with the regular Python interpeter or use the provided spark-submit as you prefer.
+If you have PySpark pip installed into your enviroment (e.g., `pip instal pyspark` you can run your application with the regular Python Interpreter or use the provided 'spark-submit' as you prefer.
 
 {% highlight bash %}
-# Use spark-submit to run your application
+# Use the Python Interpreter to run your application
```
Comment (HyukjinKwon) [inline docs/quick-start.md]: I don't think we should capitalise "interpreter".

File: docs/quick-start.md [hunk 4]
```diff
@@ -421,16 +421,15 @@ $ YOUR_SPARK_HOME/bin/spark-submit \
 Lines with a: 46, Lines with b: 23
 {% endhighlight %}
 
-If you have PySpark pip installed into your enviroment (e.g. `pip instal pyspark` you can run your application with the regular Python interpeter or use the provided spark-submit as you prefer.
+If you have PySpark pip installed into your enviroment (e.g., `pip instal pyspark` you can run your application with the regular Python Interpreter or use the provided 'spark-submit' as you prefer.
```
Comment (HyukjinKwon) [inline docs/quick-start.md]: `` (e.g., `pip instal pyspark` `` -> `` `(e.g., `pip install pyspark`) ``

File: docs/quick-start.md (hunk 2 above)
Comment (lizhaoch) [inline docs/quick-start.md]: so many new lines , i did again

**Review by srowen — APPROVED** [review]

File: docs/quick-start.md [hunk 5]
```diff
@@ -421,16 +421,15 @@ $ YOUR_SPARK_HOME/bin/spark-submit \
 Lines with a: 46, Lines with b: 23
 {% endhighlight %}
 
-If you have PySpark pip installed into your enviroment (e.g. `pip instal pyspark` you can run your application with the regular Python interpeter or use the provided spark-submit as you prefer.
+If you have PySpark pip installed into your enviroment (e.g., `pip install pyspark` you can run your application with the regular Python Interpreter or use the provided 'spark-submit' as you prefer.
```
Comment (HyukjinKwon) [inline docs/quick-start.md]: `` (e.g., `pip install pyspark` `` -> `` (e.g., `pip install pyspark`) ``

File: docs/quick-start.md [hunk 6]
```diff
@@ -421,16 +421,15 @@ $ YOUR_SPARK_HOME/bin/spark-submit \
...
 
 {% highlight bash %}
-# Use spark-submit to run your application
+# Use the Python Interpreter to run your application
```
Comment (HyukjinKwon) [inline docs/quick-start.md]: `Interpreter` -> `interpeter`. I think it is Python interpreter.

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
@lizhaoch, Could you double check https://github.com/apache/spark/pull/18722#pullrequestreview-51936276? I am repeating the same comments.

**Review by lizhaoch — COMMENTED** [review]
perfect thks Hyukjin Kwon

**Review by lizhaoch — COMMENTED** [review]
I think is ok now ,thks Hyukjin Kwon

**Review by lizhaoch — COMMENTED** [review]
comment again

File: docs/quick-start.md [hunk 7]
```diff
@@ -421,16 +421,15 @@ $ YOUR_SPARK_HOME/bin/spark-submit \
 Lines with a: 46, Lines with b: 23
 {% endhighlight %}
 
-If you have PySpark pip installed into your enviroment (e.g. `pip instal pyspark` you can run your application with the regular Python interpeter or use the provided spark-submit as you prefer.
+If you have PySpark pip installed into your enviroment (e.g., `pip install pyspark` you can run your application with the regular Python interpreter or use the provided 'spark-submit' as you prefer.
```
Comment (HyukjinKwon) [inline docs/quick-start.md]: others look good but for the last one ...

`` pip install pyspark` `` ->  `` pip install pyspark`) ``  (missing closing parenthesis).

Comment (lizhaoch) [inline docs/quick-start.md]: submit again ，you are so careful,so good

File: docs/quick-start.md [hunk 8]
```diff
@@ -421,7 +421,7 @@ $ YOUR_SPARK_HOME/bin/spark-submit \
 Lines with a: 46, Lines with b: 23
 {% endhighlight %}
 
-If you have PySpark pip installed into your enviroment (e.g., `pip install pyspark` you can run your application with the regular Python interpreter or use the provided 'spark-submit' as you prefer.
+If you have PySpark pip installed into your enviroment (e.g., `pip install pyspark`), you can run your application with the regular Python Interpreter or use the provided 'spark-submit' as you prefer.
```
Comment (HyukjinKwon) [inline docs/quick-start.md]: `interpreter` was fine ...

**Comment by HyukjinKwon** [comment]
LGTM

**Review by lizhaoch — COMMENTED** [review]
ok

**Pull request closed without merging**